### DISCUSSION TITLE
Cannot download any content on iOS app despite having needed permissions?

### DISCUSSION THREAD

**Discussion by Elekam** [jellyfin/jellyfin#12706] [Q&A] [answered]
Hello,

I am trying to add the functionality that I can download movies and series on my mobile devices for offline viewing. I know the iOS app should have that feature for quite a while now. I enabled the setting "Allow media downloads" for my user, but I still cannot download anything for some reason.

I am not aware of any additional settings that need to be configured for it to work? What could cause this?

Version of my Jellyfin App: 1.5.0 (5.5.0.0) ;Expo Version: 43.0.0
Version of my Server: 10.9.11

Thanks a lot in advance for any help :)

**Comment by romatou** [comment]
So no response still 😏
I have the same issue - only Allow media downloads is enabled in user preferences.
```
Server: 10.10.7
Jellyfin Mobile (iPadOS) 1.6.1
```

**Reply by simone-panico** [reply]
Same here

**Reply by sjorge** [reply, marked as answer]
The feature in the iOS app is still experimental so not available by default.

In settings press and hold "Jellyfin Mobile (iOS)" near the bottom to enter the hidden settings menu. Note this is very experimental so prepare for bugs and unfinished spots.
